cypress/integration/channel/message_edit_post_with_attachment_spec.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint max-nested-callbacks: ["error", 3] */
+
+describe('MM-13697 Edit Post with attachment', () => {
+    before(() => {
+        // 1. Login and go to /
+        cy.login('user-1');
+        cy.visit('/');
+    });
+
+    it('Pasted text should be pasted where the cursor is', () => {
+        // 2. Got to a test channel on the side bar
```

Comment:
This will fail in case the local dev has many channels where the town-square is out of view.  I'd suggest to add `scrollIntoView` to prevent it from happening.
```suggestion
        // 2. Got to a test channel on the side bar
        cy.get('#sidebarItem_town-square').scrollIntoView();
```
(Maybe next time, we'll add specific command for this so we'll not worry about it every time.)